docs/guided-remediation.md

```diff
@@ -0,0 +1,225 @@
+
+Selecting the "Modify lockfile in place" option will bring you to the in-place information page. From here, you can see which vulnerabilities can and cannot be resolved by this strategy. By default, every possible in-place patch will chosen to be applied. You may instead choose which subset of patches you wish to apply.
+
+![Screenshot of the interactive in-place results screen](images/guided-remediation-in-place-results.png)
+
+![Screenshot of the interactive in-place patch selection screen](images/guided-remediation-in-place-choose.png)
+
+If you wish to apply the proposed in-place patches, select the "Write" option to update your lockfile with the new dependency versions.
+
+{: .warning }
+The subcommand cannot revert the changes it makes to your lockfile. Make sure you backup your lockfile beforehand.
+
+{: .note }
+Writing these changes will not reinstall your dependencies. You'll need to run `npm ci` (or equivalent) separately.
+
+## Relock and Relax Direct Dependency Remediation
+
+Relocking recomputes your entire dependency graph based on your manifest file, taking the newest possible versions of all your required packages. This causes a large number of changes to your dependency graph, which potentially carries a larger risk of breakages.
+
+Selecting the "Relock" option will bring you to the relock information page. Here, you can see which vulnerabilities are present after relocking.
+
+In addition to relocking, it may be possible to further remove vulnerabilities by bumping the required version constraints of your direct dependencies. You may choose to apply these patches by selecting them on the relock page then choosing the "Apply pending patches" option. The dependency graph will then be recomputed and you may continue to select more options.
+
+![Screenshot of the interactive relock results screen with some relaxation patches selected](images/guided-remediation-relock-patches.png)
+
+The relaxation patches are presented in order of effectiveness, with patches that resolve the most vulnerabilities with the least amount of dependency change shown first.
+
+If you wish to apply your current relock & relaxation changes, select the "Write" option to update your manifest file with the new requirements and regenerate your lockfile (if provided).
+
+{: .warning }
+The subcommand cannot revert the changes it makes to your manifest and lockfile. Make sure you backup your files beforehand.
+
+{: .note }
+
+> The `package-lock.json` file is regenerated by first deleting the existing `package-lock.json` & `node_modules/` directory, then running `npm install --package-lock-only`. To recreate `node_modules/`, you'll need to run `npm ci` separately
+>
+> The `--relock-cmd` flag can be used to change the executed install command.
+
+## Non-Interactive Mode
+
+To enable usage in scripting and automation, we provide a non-interactive mode:
```

Comment:
We should include a sample script in the OSV-Scanner repo that we can link to which shows the automation. We can use the one we wrote before.

Reply:
Added a section on scripted usage and linked (and updated) our python script.